src/main.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import App from './App'
+import './index.css'
+
+ReactDOM.createRoot(document.getElementById('root')).render(
+  <React.StrictMode>
```

Comment:
Then here instead of calling the strictmode through react you can then: 
```suggestion
 <StrictMode>
    <App />
  </StrictMode>
```